package.json

```diff
@@ -1,27 +1,29 @@
 {
   "name": "taskcluster-lib-config",
-  "version": "0.8.8",
+  "version": "0.9.0",
   "author": "Jonas Finnemann Jensen <jopsen@gmail.com>",
-  "description": "taskcluster-lib-config",
+  "description": "Configuration loader that injects env variables into YAML",
   "license": "MPL-2.0",
   "main": "./lib/config",
   "scripts": {
-    "compile": "rm -rf lib/ .test/ && babel -sd lib/ src/ && babel -sd .test/ test/",
+    "compile": "rm -rf lib/ && babel -sd lib/ src/",
```

Comment:
Ah, I see... That makes sense.. I thought it would be enough to use the compiled lib/ but we actually want it all compiled...